stylesheets/scss/_ting.scss

```diff
@@ -252,13 +251,7 @@ body.html .action-button .form-submit:hover,
     div.update-reservations {
       .form-submit {
         &:disabled {
-          background-color: $color-disabled-grey;
-          opacity: 1;
-          @include box-shadow(0 1px 2px 0 $color-disabled-shadow);
-          &:hover {
-            background: darken($color-disabled-grey, 4);
-            opacity: 1;
-          }
+          opacity: .7;
```

Comment:
For deletion of reservations we set opacity to 0.65. Any reason why it is different here?

![mine reserveringer - kobenhavns biblioteker 2016-06-06 10-41-19](https://cloud.githubusercontent.com/assets/73966/15959951/a7948020-2efd-11e6-881d-b948771d2b02.png)